extensions/jdbc/jdbc-mariadb/runtime/src/main/java/io/quarkus/jdbc/mariadb/runtime/MariaDBAgroalConnectionConfigurer.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkus.jdbc.mariadb.runtime;
```

Comment:
runtime? why not deployment time?

Reply:
Because it's used at runtime to configure the Agroal pool.

It's a CDI bean.